Lexplorer/Pages/NFTCollection.razor

```diff
@@ -0,0 +1,174 @@
+﻿@page "/collections/{tokenAddress}"
```

Comment:
Shouldn't the page name be a little more specific?

Or maybe even put the page one level below the NFT page? That would also fix the issue that there is no corresponding icon in the navigation drawer currently.

Reply:
fixed in [d6ae50b](https://github.com/fudgebucket27/Lexplorer/pull/182/commits/d6ae50b8fbece17bb4eb6d3a37a48514b8e61c1b)